airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -4,7 +4,7 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Postgres Source Spec",
     "type": "object",
-    "required": ["host", "port", "database", "username"],
+    "required": ["host", "port", "database", "schemas", "username"],
```

Comment:
If we make this new field required, all existing Postgres connection config will become invalid. Can we make it optional, and keep the current behavior to retrieve all schemas if this parameter is missing?

Reply:
Updated. Added one more test to check that we may keep this field empty and also tested on UI manually (see attached captures in PR's description). Thanks